### ISSUE TITLE
suggestion to avoid issue when multiple modality files are in the same folder

### ISSUE THREAD

**Issue by EmmaSkarsoe** [qurit/rt-utils#51]
Hi rt_Utils authors,

Thanks for a great tool !

I've experienced some problems, when I am running the add_roi function. 
I got the following error-message: _Mask must have the same number of layers (In the 3rd dimension) as input series. Expected 130, got 129._ 

The problem arises, because my folder contains both RT structs, CT scans, RT dose and RT plan. The RT dose file do also have a 'pixel_array' attribute. 

I changed a line in the function: _load_dcm_images_from_path_

from 
_if hasattr(ds, "pixel_array"):
                    series_data.append(ds)_
to 
_if ds.Modality == 'CT':
                    series_data.append(ds)_

It solved my problem. :)

**Comment by mathiser** [comment]
This is a really good idea! Perhaps instead of hardcoding it to "CT", there could be an option which defaults to CT, but is changable to other modalities. It could even be set with SOPClassUID to avoid overlapping uses of Modality?

**Comment by asim-shrestha** [comment]
Thanks for reaching out @EmmaSkarsoe, and good call @mathiser. I'm not sure when I can work on that but that seems like a good solution.

**Comment by mathiser** [comment]
Dear @asim-shrestha 
It's so great there is some activity on the repo again! 

Are you open to pull requests?

**Comment by asim-shrestha** [comment]
Yes of course! Please feel free to submit any of your desired changes :)
